tests/integration_tests/without_db/udf_communication/peer_communication/test_add_peer_and_wait_close.py

```diff
@@ -28,8 +28,8 @@
     logger_factory=WriteLoggerFactory(file=Path(__file__).with_suffix(".log").open("wt")),
     processors=[
         structlog.contextvars.merge_contextvars,
-        ConditionalMethodDropper(method_name="debug"),
-        ConditionalMethodDropper(method_name="info"),
+        #ConditionalMethodDropper(method_name="debug"),
+        #ConditionalMethodDropper(method_name="info"),
```

Comment:
Did you comment that out during some debugging maybe?

Reply:
yep, enabled it again